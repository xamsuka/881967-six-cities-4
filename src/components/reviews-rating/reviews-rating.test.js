import React from 'react';
import renderer from 'react-test-renderer';
import ReviewsRating from './reviews-rating.jsx';


test(`<ReviewsRating /> render`, () => {
  const three = renderer
    .create(<ReviewsRating />).toJSON();

  expect(three).toMatchSnapshot();
});
