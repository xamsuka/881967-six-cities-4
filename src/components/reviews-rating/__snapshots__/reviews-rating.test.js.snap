// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReviewsRating /> render 1`] = `
<div
  className="reviews__rating rating"
>
  <div
    className="reviews__stars rating__stars"
  >
    <span
      style={
        Object {
          "width": "50%",
        }
      }
    />
    <span
      className="visually-hidden"
    >
      Rating
    </span>
  </div>
</div>
`;
